### ISSUE TITLE
Add full bleed illustration option to hero graphic guidelines

### ISSUE THREAD

**Issue by nataliafitzgerald** [cfpb/design-manual#462]
## Add full bleed illustration option to hero graphic guidelines

**User story**
Updates to hero graphic guidelines #461

### Description
The full bleed illustration hero option has been available to users (and in use) for quite some time but the Design Manual does not include this option. 

Full bleed illustrated hero: 
https://www.consumerfinance.gov/consumer-tools/prepaid-cards/

### Task
Add full bleed illustration option to hero graphic guidelines
- [X] Develop usage guidelines
- [X] Create supporting images
- [X] Update Design Manual

#### Ready Checklist
- [X] Has acceptance criteria or is part of a user story
- [X] Describes the smallest possible chunk of work
- [X] There are no blockers to starting the task
- [X] All information necessary for completion is available 
      - All input from other disciplines (Design/Ux/Content/FEWD/BEWD) is attached
      - There are no outstanding questions for stakeholders

#### Acceptance criteria
- [X] Usage guidelines are developed
- [X] Supporting images are created
- [X] Design Manual is updated

#### Done Checklist
- [X] All work has been peer-reviewed
- [X] Has been acceptance tested and meets acceptance criteria
- [X] All code or assets are merged to master
- [X] Documentation has been updated.

**Comment by nataliafitzgerald** [comment]
I've been working on this update as part of the overall hero page updates. 

Here's the content related to bleed heroes:

## Illustration (bleed)
- On large screens, illustration bleeds top and bottom
- Small and large screen images should contain the same elements but can differ compositionally
- Large screen image dimensions are based on maximum possible hero height at 601px width
- To conserve vertical space, small screen image should be as short as possible
- Illustration dimensions for large screens (exact): 470px x 638px (2x: 940px x 1276px)
- Illustration dimensions for small screens: 570px (exact) x 320px (maximum) (2x: 1140px x 640px) 

**Large screens (601+)**
![hero_style_variations_bleed_large](https://cloud.githubusercontent.com/assets/6887128/24635483/a212e91e-18a1-11e7-95b1-0a8952778ef3.png)

**Small screens (600-)**
![hero_style_variations_bleed_small](https://cloud.githubusercontent.com/assets/6887128/24635485/a70137d2-18a1-11e7-9483-1386b4b40414.png)

**Comment by jimmynotjim** [comment]
@Scotchester Is 638px the tallest the hero is in height before breaking the image below the text? That seems wicked tall.

**Comment by Scotchester** [comment]
@jimmynotjim Correct. The 638 figure is a little annoying to calculate, but it's what the max height of the hero image should be on a 601px viewport, with the longest possible copy (i.e., tallest total hero space).

I won't spell it out further, because the current implementation has a width bug in it, which makes explaining the calculation cumbersome.

**Comment by nataliafitzgerald** [comment]
@jimmynotjim 
Yeah, here's the bullet that I included to try to explain it on the page: "Large screen image dimensions are based on maximum possible hero height at 601px width". 

The maximum height was determined by using the maximum height that would be created at the 601px width - based on the content guidelines. Here's the image where I tried to figure it out in illustrator.
<img width="658" alt="screen shot 2017-04-04 at 6 01 11 pm" src="https://cloud.githubusercontent.com/assets/6887128/24680916/b887649e-1960-11e7-818b-a27bb3ebd405.png">
